howso/direct/core.py

```diff
@@ -581,8 +539,7 @@ def clean_data(
         remove_duplicates : bool, default False
             If true, will remove duplicate cases (cases with identical values).
         """
-        return self._execute("clean_data", {
-            "trainee": trainee_id,
+        return self._execute(trainee_id, "clean_data", {
```

Comment:
`trainee_id` is typed as optional in this method, the type should be updated since it is required.

Reply:
dropped the "optional" from docstring and type hint, good eye.